patches/0004-Add-OpenSSL-crypto-backend.patch

```diff
@@ -226,26 +226,36 @@ index 00000000000000..9995444d1f1e9f
 +// Fedora derived distros use different naming for the version 1.0.x.
 +var knownVersions = [...]string{"3", "1.1", "11", "111", "1.0.2", "1.0.0", "10"}
 +
++const lcryptoPrefix = "libcrypto.so."
++
 +func init() {
-+	version, _ := syscall.Getenv("GO_OPENSSL_VERSION_OVERRIDE")
-+	if version == "" {
-+		var fallbackVersion string
++	var lcrypto string
++	if version, _ := syscall.Getenv("GO_OPENSSL_VERSION_OVERRIDE"); version != "" {
```

Comment:
I had to update this code because `openssl.Init` now accepts a file path instead of a version suffix.